test/utils_spec.js

```diff
@@ -1,64 +1,294 @@
 import moment from 'moment';
```

Comment:
Added a lot of tests to make sure that the core time functionality returns the correct data given different input permutations.